common/schema-builder/src/main/java/io/smallrye/graphql/schema/SchemaBuilder.java

```diff
@@ -147,7 +148,8 @@ private List<MethodInfo> getAllMethodsIncludingFromSuperClasses(ClassInfo classI
         IndexView index = ScanningContext.getIndex();
         List<MethodInfo> methods = new ArrayList<>();
         while (current != null) {
-            methods.addAll(current.methods());
+            methods.addAll(
+                    current.methods().stream().filter(methodInfo -> !methodInfo.isSynthetic()).collect(Collectors.toList()));
```

Comment:
Just for the record, this would be simpler and probably more efficient:

```java
current.methods().stream().filter(methodInfo -> !methodInfo.isSynthetic()).forEach(methods::add);
```

(Of course avoiding streams would be even more efficient, but that wouldn't be a one-liner anymore :-) )